pkg/printers/events/formatter.go

```diff
@@ -129,19 +128,23 @@ func (ef *formatter) FormatActionGroupEvent(
 	if age.Action == event.PruneAction &&
 		age.Type == event.Finished &&
 		list.IsLastActionGroup(age, ags) {
+		ps := s.PruneStats
 		ef.print("%d resource(s) pruned, %d skipped, %d failed", ps.Pruned, ps.Skipped, ps.Failed)
 	}
 
 	if age.Action == event.DeleteAction &&
 		age.Type == event.Finished &&
 		list.IsLastActionGroup(age, ags) {
-		ef.print("%d resource(s) deleted, %d skipped", ds.Deleted, ds.Skipped)
+		ds := s.DeleteStats
+		ef.print("%d resource(s) deleted, %d skipped, %d failed", ds.Deleted, ds.Skipped, ds.Failed)
```

Comment:
do we want to say "failed to delete"? "failed" by itself feels a little ambiguous. same with failed to reconcile below.

Reply:
fixed.